src/live/customize.rs

```diff
@@ -25,6 +25,9 @@ use crate::iso9660::{self, IsoFs};
 use super::embed::{INITRD_IGNITION_PATH, INITRD_NETWORK_DIR};
 use super::util::filename;
 
+use nmstate::NetworkState;
+use serde_json;
```

Comment:
Let's put these in the first block with the other crate imports.

Reply:
Done